zeebe/gateway-grpc/src/main/java/io/camunda/zeebe/gateway/EndpointManager.java

```diff
@@ -493,17 +490,11 @@ private <GrpcRequestT, BrokerResponseT> BrokerRequest<BrokerResponseT> mapToBrok
 
     final BrokerRequest<BrokerResponseT> brokerRequest = requestMapper.apply(grpcRequest);
 
-    final List<String> authorizedTenants =
-        multiTenancy.isEnabled()
-            ? Context.current().call(InterceptorUtil.getAuthorizedTenantsKey()::get)
-            : List.of(TenantOwned.DEFAULT_TENANT_IDENTIFIER);
-
     final var authorizationToken =
         Authorization.jwtEncoder()
             .withIssuer(JwtAuthorizationBuilder.DEFAULT_ISSUER)
             .withAudience(JwtAuthorizationBuilder.DEFAULT_AUDIENCE)
-            .withSubject(JwtAuthorizationBuilder.DEFAULT_SUBJECT)
-            .withClaim(Authorization.AUTHORIZED_TENANTS, authorizedTenants);
+            .withSubject(JwtAuthorizationBuilder.DEFAULT_SUBJECT);
 
     // retrieve the user claims from the context and add them to the token
```

Comment:
❓ One question I've got regarding this is how will this work with Basic auth? It seems the current `AuthenticationInterceptor` is only suited to deal with oauth. I'd expect we need to set the user key on the authorizations in that case. Similar to what we do in REST in the `RequestMapper`:

`token.withClaim(Authorization.AUTHORIZED_USER_KEY, authenticatedUserKey);`

Reply:
As much as I hate to say it, if we go that route I'd rather work with username for the REST gateway as well. That way we at least keep them aligned.

Reply:
I'm not sure what's the impact of changing the behaviour, it might be too much effort at the moment

Reply:
> As much as I hate to say it, if we go that route I'd rather work with username for the REST gateway as well. That way we at least keep them aligned.

@remcowesterhoud I like the idea of keeping the gateways aligned. My only concern is that we will add an extra read from the Zeebe State, and we generally try to reduce State reads/lookups. 

If we replace the `userKey` with a `username` in the REST API gateway, that means an additional state lookup will be always performed with every request. 

If we don't align them, the "username workaround" will remain on gRPC only. With the deprecation of most of the gRPC calls, an additional state lookup will only be performed for job-related gRPC calls. It might be more optimal at the cost of alignment.